### ISSUE TITLE
error NU5125 when compiling the library

### ISSUE THREAD

**Issue by MagicAndre1981** [aloneguid/config#117]
You still use License and Icon URLs, which should be avoided and replaced by [PackageLicenseExpression](https://docs.microsoft.com/en-us/nuget/reference/msbuild-targets#packing-a-license-expression-or-a-license-file) element:

```
<PropertyGroup>
    <PackageLicenseExpression>MIT</PackageLicenseExpression>
</PropertyGroup>
```

and [PackageIcon file](https://docs.microsoft.com/en-us/nuget/reference/msbuild-targets#packing-an-icon-image-file).

My RP #116 only compiles if `TreatWarningsAsErrors` is set to false, but I wanted to let you make the final decision what to do to solve this.

**Comment by MagicAndre1981** [comment]
any update @aloneguid ?

**Comment by aloneguid** [comment]
Sorry i've been busy for some time with python/scala/c++ and... rust. I'm back now and promise to be more active ;)
Rebased and released in 4.17.0.

**Comment by aloneguid** [comment]
All migrated to github actions and packaging fixed according to .net sdk 5. I hope I did it right.

**Comment by MagicAndre1981** [comment]
Yes works better, now I only get the error NU5026 because dlls are not found during build from cli:

> C:\Program Files\dotnet\sdk\5.0.401\Sdks\NuGet.Build.Tasks.Pack\buildCrossTargeting\NuGet.Build.Tasks.Pack.targets(
221,5): error NU5026: Die im Paket einzubeziehende Datei "config\src\Impl\Azure\Config.Net.Azure.KeyVault\b
in\Release\netstandard2.0\Config.Net.Azure.KeyVault.dll" wurde auf dem Datenträger nicht gefunden. 
C:\Program Files\dotnet\sdk\5.0.401\Sdks\NuGet.Build.Tasks.Pack\buildCrossTargeting\NuGet.Build.Tasks.Pack.targets(
221,5): error NU5026: Die im Paket einzubeziehende Datei "config\src\Config.Net.Json\bin\Release\net5.0\Con
fig.Net.Json.dll" wurde auf dem Datenträger nicht gefunden. [config\src\Config.Net.Json\Config.Net.Json.csp
roj]

when I compile the missing projects on my own before building config.net I no longer get any error.

Btw, can you add .net 6 support (#120)?

**Comment by aloneguid** [comment]
ok this is odd because github actions builds it from a clean slate with no issues - https://github.com/aloneguid/config/actions/runs/1243269281. Can it be something with your environment? Could you try to do a clean clone and build just to make sure please?

**Comment by MagicAndre1981** [comment]
> Could you try to do a clean clone and build just to make sure please?

I did this and get the same error when I run `dotnet pack -c Release  /p:PackageVersion=4.17.1-beta /p:Version=4.17.1-beta /p:AssemblyVersion=4.17.1 /p:FileVersion=4.17.1`

I had to run `dotnet build -c Release  /p:PackageVersion=4.17.1-beta /p:Version=4.17.1-beta /p:AssemblyVersion=4.17.1 /p:FileVersion=4.17.1` in first step. So the pack command skips the compile of this 1 project and later complains about missing file. Wired.

In your script, you still use the old 5.0.100 SDK, you update to latest version 5.0.401, otherwise you get warnings with the DotNet.ReproducibleBuilds I've added in my RP.
